hphp/runtime/vm/jit/vasm-instr.h

```diff
@@ -890,6 +894,11 @@ struct srem { Vreg s0, s1, d; };
  */
 struct divint { Vreg s0, s1, d; };
 
+/*
+ * Integer multiplication.
+ */
+struct mulint { Vreg s0, s1, d; Vlabel targets[2]; };
```

Comment:
Document that this is multiplication with overflow detection, and what the branches mean; it's definitely not clear without any explanation.

Also, probably rename `mulint{}` to `muloint{}` or `mulinto{}` or `mulo{}` or something.